x-pack/plugins/infra/public/utils/get_chart_theme.ts

```diff
@@ -1,40 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { Theme, PartialTheme, LIGHT_THEME, DARK_THEME, SettingsProps } from '@elastic/charts';
-
-// TODO use the EUI charts theme see src/plugins/charts/public/services/theme/README.md
-export function getChartTheme(isDarkMode: boolean): Theme {
-  return isDarkMode ? DARK_THEME : LIGHT_THEME;
-}
-
-const TIMELINE_LIGHT_THEME: PartialTheme = {
-  crosshair: {
-    band: {
-      fill: '#D3DAE6',
-    },
-  },
-  axes: {
-    gridLine: {
-      horizontal: {
-        stroke: '#eaeaea',
```

Comment:
This was done to hide the vertical lines. we can achieve the same by passing `visible: false`

<img width="1143" alt="image" src="https://github.com/elastic/kibana/assets/2767137/c92e56c4-4e84-4cbc-8796-fd9c0a436b14">